front/admin/cli.ts

```diff
@@ -567,7 +567,9 @@ const eventSchema = async (command: string, args: parseArgs.ParsedArgs) => {
       if (!args.properties) {
         throw new Error("Missing --properties argument");
       }
+      const sId = new_id();
       const schema = await EventSchema.create({
+        sId: sId.slice(0, 10),
```

Comment:
kind of nit, should we have a `size` optional param to this `new_id` function ? also, should it be `newId` instead ?

Reply:
Agreed about the NIT -> I even think it should not be a param but instead a `generateModelSid` that returns a 10 char sID, to enforce having always the same sId for models. I would do it in another PR so, as we need to refactor all places where we create an object that has an sID. OK for you?

Reply:
Yes of course !

Reply:
Done here: https://github.com/dust-tt/dust/pull/1162